### PR TITLE
Components

### PR DESCRIPTION
**Pull request by peterdgreat**
# I developed a React component that will hold the core functionality of the Calculator
- Created a class component for button
- Import it to the main app
- Styled all components according to design

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @peterdgreat, 

**This is a great project you have going on, well-done lad**. 

   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️

- [ ] Remove all the react boiler-plate files that you are not using in your app. (reportwebvitals, apptest, setuptests)

_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
```
Comment (mke2111) [inline src/App.js]: - [ ] Instead of using a functional component, use a class-based component for this file. This is one of the project's requirements.

Comment (mke2111) [inline src/App.js:5]: - [ ] Instead of using a functional component, use a class-based component for this file. This is one of the project's requirements.

Comment (peterdgreat) [reply src/App.js]: ooh thanks, I totally forgot about that

**Review by dasileker — APPROVED** [review]
## **Hi @peterdgreat,** 

## **Congratulations! 🎉 Project Approved ✅**
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 

Approved

 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**